jdaviz/configs/default/plugins/collapse/collapse.vue

```diff
@@ -34,7 +24,7 @@
       ></v-select>
     </v-row>
 
-    <div v-if="selected_axis==0">
```

Comment:
If we're getting rid of the axis option entirely, then the `v-else` following this should also be removed (I see you already removed the `add_replace_results` traitlet that it uses.

Reply:
Good catch! Done.